packages/ckeditor5-ui/theme/components/dropdown/toolbardropdown.css

```diff
@@ -7,16 +7,14 @@
 	--ck-toolbar-dropdown-max-width: 60vw;
 }
 
-.ck.ck-toolbar-dropdown {
-	& .ck-dropdown__panel {
-		/* https://github.com/ckeditor/ckeditor5/issues/5586 */
-		width: max-content;
-		max-width: var(--ck-toolbar-dropdown-max-width);
+.ck.ck-toolbar-dropdown > .ck-dropdown__panel {
```

Comment:
FYI @oleq I specified this selector because it affected the size of dropdowns nested in the toolbar dropdown (it was messing up with the sizes of the panels).